crates/rolldown/tests/esbuild/default/external_with_wildcard/artifacts.snap

```diff
@@ -16,3 +16,27 @@ snapshot_kind: text
 ────╯
 
 ```
+## UNRESOLVED_IMPORT
+
+```text
+[UNRESOLVED_IMPORT] Warning: Could not resolve "/dir/file.gif" in entry.js
+   ╭─[entry.js:9:8]
+   │
+ 9 │ import "/dir/file.gif";
+   │        ───────┬───────  
+   │               ╰───────── Module not found.
+───╯
+
+```
+## UNRESOLVED_IMPORT
+
+```text
+[UNRESOLVED_IMPORT] Warning: Could not resolve "/sassets/images/test.jpg" in entry.js
+   ╭─[entry.js:8:8]
+   │
+ 8 │ import "/sassets/images/test.jpg";
```

Comment:
Esbuild has this behavior too. https://github.com/evanw/esbuild/blob/a08f30db4a475472aa09cd89e2279a822266f6c7/internal/bundler_tests/bundler_default_test.go#L2571-L2601